### DISCUSSION TITLE
Local compile error of Doris 2.0.2-rc05

### DISCUSSION THREAD

**Discussion by zhaofei1193** [apache/doris#27008] [RFC]
下载doris的2.0.2-rc05的源码后，本地编译报错：org.apache.doris:je:pom:18.3.14-doris-SNAPSHOT包拉取不到

<dependency>
                <groupId>org.apache.doris</groupId>
                <artifactId>je</artifactId>
                <version>${je.version}</version>
            </dependency>

**Comment by linyy1991** [comment]
遇到同样的问题，解决了吗？

**Comment by zhaofei1193** [comment]
放弃了，我搞不定



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年1月13日(周六) 中午11:58
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [apache/doris] 下载doris的2.0.2-rc05的源码后，本地编译报错：org.apache.doris:je:pom:18.3.14-doris-SNAPSHOT包拉取不到 (Discussion #27008)




 
遇到同样的问题，解决了吗？
